### ISSUE TITLE
monoallelic_X_hemizygous 

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7496]
**Preferred term label:**
monoallelic_X_hemizygous 

**Synonyms**
X-linked Recessive inheritance
HP:0001419

**Definition (free text, please give PubMed ID)**
The allelic requirement for  X-linked Recessive inheritance

**Parent term (use hpo.jax.org/app)**
allelic requirement (requested)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as allelic requirement synonym
